.k8s/deploy.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: chord-fe-deploy
+  labels:
+    app: chord-fe
+spec:
+  replicas: 25
+  selector: 
+    matchLabels:
+      app: chord-fe
+  template:
+    metadata:
+      labels:
+        app: chord-fe
+    spec:
+      containers:
+      - name: chord-fe
+        image: pw1124/chord:latest
```

Comment:
does this automatically pull from dockerhub?

Reply:
yes, but if you try to deploy it on your own cluster you need to setup a credential in your local k8s called `regcred` that contains your dockerhub login